zhaquirks/philips/lca003.py

```diff
@@ -0,0 +1,111 @@
+"""Quirk for Phillips LCA003."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice, CustomCluster
+from zigpy.zcl.clusters.general import (
+    OnOff,
+    Basic,
+    Identify,
+    LevelControl,
+    Scenes,
+    Groups,
+    Ota,
+    GreenPowerProxy,
+)
+
+import zigpy.types as t
+from zigpy.zcl.clusters.lighting import Color
+from zigpy.zcl.clusters.lightlink import LightLink
+
+from zhaquirks.const import (
+    ENDPOINTS,
+    OUTPUT_CLUSTERS,
+    INPUT_CLUSTERS,
+    DEVICE_TYPE,
+    PROFILE_ID,
+    MODELS_INFO,
+)
+from zhaquirks.philips import PHILIPS
+
+
+class PowerOnState(t.enum8):
+    """Philips power on state enum."""
+
+    Off = 0x00
+    On = 0x01
+    LastState = 0xFF
+
+
+class PhilipsOnOffCluster(CustomCluster, OnOff):
+    """Philips OnOff cluster."""
+
+    attributes = OnOff.attributes.copy()
+    attributes.update({0x4003: ("power_on_state", PowerOnState)})
```

Comment:
move these to init.py in the same module so you can import them into both files and remove the code duplication